plugins/tokens/update.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = joi.reach(schema.models.token.base, 'id');
+
+module.exports = () => ({
+    method: 'PUT',
```

Comment:
Today I learned that there are http methods other than `GET`, `POST`, `PUT`, and `DELETE`, and that `PUT` requires the full resource be in the request.